Formula/confluent-hub-client.rb

```diff
@@ -0,0 +1,13 @@
+class ConfluentHubClient < Formula
+  desc "Confluent Hub client"
+  homepage "https://confluent.io"
+  url "https://github.com/confluentinc/hub-client/releases/download/v5.1.0-beta180619025141/confluent-hub-client-5.1.0-beta180619025141-package.tar.gz", :using => GitHubPrivateRepositoryReleaseDownloadStrategy
```

Comment:
Do we want to use 5.1.0, or should we use 5.0.0?

Reply:
How does this work with the private repo? Will non-Confluent people be able to get this? The following:

    $ curl https://github.com/confluentinc/hub-client/releases/download/v5.1.0-beta180619025141/confluent-hub-client-5.1.0-beta180619025141-package.tar.gz

results in `Not Found`.

Reply:
I've just realized, that `GitHubPrivateRepositoryReleaseDownloadStrategy` uses `HOMEBREW_GITHUB_API_TOKEN` to download files from private repos, so people outside of Confluent wouldn't be able to install it. So I've created a public S3 bucket to host file there.

How about versioning of the client? Do we want to use the same versions we use for CP?
I've used `v5.1.0-beta180619025141` since it has an important fix.